hawk/app/views/cib/meta.html.haml

```diff
@@ -0,0 +1,28 @@
+.modal-header
+  %button.close{ type: "button", data: { dismiss: "modal" } }
+    %span{ aria: { hidden: "true" } }
+      &times;
+    %span.sr-only
+      = _("Close")
+
+  %h3.modal-title
+    = icon_text "info", _("Cluster Details"), class: "page"
+
+.modal-body
+  - unless current_cib.live?
+    .alert.alert-warning{role: :alert}
+      = _("Simulator Active")
+
+  %table.table.table-bordered.metadata
+    %tbody
+      - footer_metadata_items.each do |name, link, value, title|
+        %tr
+          %th.col-xs-4
+            - if title
+              %abbr{title: title}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace